### DISCUSSION TITLE
module 'matplotlib' has no attribute 'colormaps'

### DISCUSSION THREAD

**Discussion by hamediut** [marcomusy/vedo#1157] [General]
Hello, I am trying to use vedo to make 3D time lapse videos from 3D tif images I have. but I get an error about matplotlib colormap. which should be somewhere in Video module in vedo. can you help me to fix this error. I thought it may be due to the cinflict between matplotlib and vedo versio, so I updated both but didn't help. 
--------------------------------------------------------------
Here are my package versions:
vedo -> 2024.5.2
vtk  -> 9.3.1
matplotlib -> 3.4.3
python  ->  3.9.7
--------------------------------------------------------------
Here is the code I run:
![image](https://github.com/user-attachments/assets/9c0176d1-3a64-4eb8-a749-d4d0004746d9)

--------------------------------------------------------------
And here is the error I get:
![image](https://github.com/user-attachments/assets/19a15bca-037c-468e-a7ac-af20648d1b76)

**Comment by marcomusy** [comment]
I think you need matplotlib version v3.5 at least, if you cannot update it you can try the vedo dev version:
```
pip install -U git+https://github.com/marcomusy/vedo.git
```
